### PR TITLE
Feat/generate keypairs

### PR DESCRIPTION
**Pull request by mennatabuelnaga**
- Generates a seed secret key and saves it in `seda_secret_key`
- Creates a `crypto` module adding two fns `derive_bn254_key_pair(mnemonic_path: &str, index: usize)` and `derive_ed25519_key_pair(mnemonic_path: &str, index: usize)`
- Adds two tests: generate_bn254_pair() and generate_ed25519_pair()

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,10 +1,11 @@
 [workspace]
 resolver = "2"
-default-members = ["cli"]
+default-members = ["cli", "crypto"]
```
Comment (mariocao) [inline Cargo.toml:3]: Why is it a default member?

File: Cargo.toml [hunk 2]
```diff
@@ -42,14 +43,17 @@ opt-level = "z"
 
 [workspace.dependencies]
 actix = "0.13"
+anyhow = "1.0"
```
Comment (mariocao) [inline Cargo.toml:46]:
```suggestion
```

File: Cargo.toml (hunk 1 above)
Comment (mennatabuelnaga) [reply Cargo.toml]: I just did it because it's easier for me to build along with cli for manual testing purposes but I forgot to revert it

**Review by mariocao — APPROVED** [review]

**Review by FranklinWaller — APPROVED** [review]

**Review by gluax — APPROVED** [review]
Can you regenerate the CLI.md file, please, before merging?

**Review by gluax — APPROVED** [review]

**Review by mariocao — APPROVED** [review]

**Pull request merged**